### DISCUSSION TITLE
Bulk renaming Start and End location

### DISCUSSION THREAD

**Discussion by Thompson-ongithub** [mytourbook/mytourbook#1409] [Q&A]
Is there a easy way to bulk rename the Start and End location fields?
The reason is that with the (rather) new feature of automatic Start / End locations the ways those locations are noted are different from when I inserted them by hand, leaving me now with quite a lot of double entries...

Thanks.

**Comment by wolfgang-ch** [comment]
The tour start/end locations can be set for multiple selected tours, is this not enough?

![image](https://github.com/user-attachments/assets/f0ee5fe3-7f88-4213-b4d5-665988b7ad73)

**Comment by Thompson-ongithub** [comment]
Thanks for the quick answer.
If I made no mistake, your suggestion works well for all Tours that have a GPS track, but I was not able to bulk set Start / End locations for other tours, not based on GPS tracks (i.e. HAK-Tours, Polar Tour etc.)

**Reply by wolfgang-ch** [reply]
> If I made no mistake, your suggestion works well for all Tours that have a GPS track, but I was not able to bulk set Start / End locations for other tours, not based on GPS tracks (i.e. HAK-Tours, Polar Tour etc.)

this is correct only GPS tracks are currently supported in this feature because it needs a position to get the location informations.

Can you create a feature request for bulk start/end location setting which will be just a simple text field to enter any

**Comment by wolfgang-ch** [comment]
I create a feature request https://github.com/mytourbook/mytourbook/issues/1413
